tm2/pkg/sdk/events.go

```diff
@@ -35,3 +38,63 @@ func (em *EventLogger) EmitEvents(events []Event) {
 // ----------------------------------------------------------------------------
 
 type Event = abci.Event
+
+func NewEvent(eventType string, pkgPath string, ident string, timestamp int64, attrs ...EventAttribute) Event {
+	return AttributedEvent{
```

Comment:
I think we should move this to [gnovm/stdlibs/std/emit_event.go](https://github.com/gnolang/gno/pull/1653/files#diff-8a4ae96a0ef873192a57572d169e19f024acb09290c0efd05fb3c7cdab0f11bf)

Reply:
Or consider renaming things to avoid confusion.

Reply:
renamed to `NewDetailedEvent`